frontend/src/metabase/css/core/base.styled.ts

```diff
@@ -2,12 +2,8 @@ import type { Theme } from "@emotion/react";
 import { css } from "@emotion/react";
 
 export const getRootStyle = (theme: Theme) => css`
-  --default-font-size: 0.875em;
```

Comment:
I inline this instead since there's no point exposing this custom property if it's only used in one place below.